### PR TITLE
feat: provide remote IP address to external auth script

### PR DESCRIPTION
**Pull request by kklash**
# Summary

This PR provides the remote client IP address to the auth script under the `--remote-ip` flag.

The auth script can use this to track abuse and apply rate-limiting using external mechanisms.

### CODE REVIEW

File: src/device/mod.rs [hunk 1]
```diff
@@ -852,6 +852,8 @@ fn check_auth (hh: &handshake::HalfHandshake, d: &mut LockReadGuard<Device>) {
                         .arg("wg")
```
Comment (clukawski) [inline src/device/mod.rs:852]:
```suggestion
                        .arg("wg")
                        .arg("up")
```

So we'll actually need this as well.

Comment (clukawski) [reply src/device/mod.rs:852]: Might as well get this in in the same PR.

Comment (kklash) [reply src/device/mod.rs:852]: Nice catch. This might explain why wireguard hasn't been functional lately on Freshscribe

**Comment by clukawski** [comment]
I can't review this but this looks perfectly fine to me.

**Pull request merged**